app/client/components/history/historyDrawer.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Drawer, Timeline } from "antd";
+import { connect } from "react-redux";
+import { toTitleCase, replaceUnderscores } from "../../utils";
+interface Props {
+  drawerVisible: boolean;
+  onClose: any;
+  changes: [];
+}
+
+class HistoryDrawer extends React.Component<Props> {
+  constructor(props: any) {
+    super(props);
+  }
+
+  render() {
+    const stringList = this.props.changes.map(change => {
+      return `${toTitleCase(
+        replaceUnderscores(change.column_name)
+      )} changed from "${change.old_value}" to "${change.new_value}" by ${
+        change.change_by.email
+      }`;
+    });
+    return (
+      <Drawer
+        title="Account History"
+        placement="right"
+        visible={this.props.drawerVisible}
+        onClose={this.props.onClose}
+        width={500}
+      >
+        <Timeline>
+          {stringList.map(item => (
+            <Timeline.Item>{item}</Timeline.Item>
```

Comment:
This item needs a key
<img width="582" alt="Screen Shot 2021-12-15 at 4 11 41 pm" src="https://user-images.githubusercontent.com/27039316/146127366-ebfe8369-576a-44b7-9985-089a00c7069e.png">